### ISSUE TITLE
Create single report for Django

### ISSUE THREAD

**Issue by rvnovaes** [xmlrunner/unittest-xml-reporting#44]
Hello to all!

Is is possible to generate a single report for a django project. For instance, let's suppose our django project has 2 apps: app1 and app2. When executing python manage.py test app1 app2 I would like to have all the output to a single file, instead of the two that are generated. The reason is that the sonarqube python plugin expects only one nosetests.xml file. It would be nice to have it as a setting on settings.py. For instance:
TEST_OUTPUT_TO_SINGLE_FILE = true or false
TEST_OUTPUT_FILE_NAME = nosetests.xml

**Comment by dnozay** [comment]
see pull #76, `TEST_OUTPUT_DIR = 'nosetests.xml'`

**Comment by benjaminrigaud** [comment]
I just tried with Django 1.8 and unittest-xml-reporting==1.12.0
It generates a nosetests.xml directory containing all the xml files

**Comment by ZuluPro** [comment]
I just tried with Python 2.7, Django 1.7/1.8 and unittest-xml-reporting==1.12.0.
I launched:
```
TEST_OUTPUT_DIR=junit.xml python3 myblog/manage.py test
```

After tests I have an XML report for each testcase:
```
ls TEST-myblog.*
TEST-myblog.about.tests.ViewAboutMeTest-20150920124528.xml  TEST-myblog.myapp.tests.BlogTest-20150920124528.xml
TEST-myblog.myapp.tests.AdminTest-20150920124528.xml        TEST-myblog.myapp.tests.CommandSettingsTest-20150920124528.xml
```

I'm with @rvnovaes, we should have the ``TEST_OUTPUT_TO_SINGLE_FILE``
But ``TEST_OUTPUT_DIR`` doesn't seem to work too:
```
TEST_OUTPUT_DIR=/tmp/dir myblog/manage.py test
ls /tmp/dir
# (empty)
```

**Comment by dnozay** [comment]
Should be fixed with ad1a3fca364fbd82d0fc634e6e6dda3665982594, but don't take my word for it.
